src/collections/integrations/devfile/index.mdx

```diff
@@ -15,8 +15,8 @@ featureList: [
   "Central location management so updates can be applied once and be properly aligned across development teams."
 ]
 workingSlides: [
-  ../_images/meshmap-visualizer.png,
-  ../_images/meshmap-designer.png
+  ../_images/meshmap-designer.png,
```

Comment:
@narasimha-1511  aren't the images supposed to be inside `_images/devfile` directory

Reply:
they are inside devfile folder these images are automatically used in the integration from the folder you can see it here:https://65cc7c304419d787fb0f8004--layer5.netlify.app/cloud-native-management/meshery/integrations/devfile